src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.classes.figures.Figure;
+import core.basesyntax.classes.functional.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.